recipes/kafka-connect.rb

```diff
@@ -0,0 +1,101 @@
+include_recipe "confluent::_install"
+
+confluent_extracted_dir = File.join(node["confluent"]["install_dir"], "confluent-#{node["confluent"]["version"]}")
+connect_jdbc_jar_dir = File.join confluent_extracted_dir, 'share/java/kafka-connect-all'
+
+directory "#{confluent_extracted_dir}/etc/kafka-connect" do
+  owner node["confluent"]["user"]
+  group node["confluent"]["group"]
+  mode "755"
+  action :create
+end
+
+link "/etc/kafka-connect" do
+  to "#{confluent_extracted_dir}/etc/kafka-connect"
+end
+
+# Drops in any jars a remote url was provided for
+node["confluent"]["kafka-connect"]["jar_urls"].each do |kafka_connect_jar_url|
```

Comment:
the bin files for distributed and standalone connect have this code.
```
java_base_dir="$base_dir/../share/java"
# confluent-common: required by kafka-serde-tools
# kafka-serde-tools (e.g. Avro serializer): bundled with confluent-schema-registry package
for library in "confluent-common" "kafka-serde-tools"; do
  CLASSPATH=$CLASSPATH:$java_base_dir/$library/*
done
# Classpath addition for any Kafka Connect connectors
for library in $java_base_dir/kafka-connect-*; do
  CLASSPATH=$CLASSPATH:$library/*
done
```
so anything dropped in that share directory with `kafka-connect-*` is picked up. I thought adding it there would be more consistent with the way confluent was doing it